sopel/plugins/handlers.py

```diff
@@ -540,8 +542,26 @@ def get_meta_description(self) -> PluginMetaDescription:
         })
         return data
 
+    def get_version(self) -> str | None:
+        """Retrieve the plugin's version.
+
+        :return: the plugin's version string
+
+        If the plugin module doesn't have a ``__version__`` attribute, this
+        method uses the file's last modification date as the version string.
+        """
+        version = super().get_version()
+        if version is None and self.file_mtime is not None:
+            # try to get version from file metadata
+            dt = datetime.fromtimestamp(self.file_mtime)
+            version = dt.strftime("%Y.%m.%d")
```

Comment:
Nitpick: it feels a little weird to me to call that "getting" rather than constructing the default version. I guess maybe "try" is throwing me since I can't see any reason these operations might fail with the other changes.